demo-app/client/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
```

Comment:
Is this file needed, i.e. do we want to keep the settings of the demo app independent?

Reply:
removed